benchmarks/executor.py

```diff
@@ -1,11 +1,21 @@
 # stdlib
+import os
+from pathlib import Path
 import subprocess
+from typing import Dict
+from typing import List
+from typing import Optional
+from typing import Tuple
 
 # third party
-import numpy as np
 import pyperf
-from syft_benchmarks import run_rept_suite
-from syft_benchmarks import run_sept_suite
+from syft_benchmarks import run_ndept_suite
+from syft_benchmarks import run_rept_suite  # noqa: F401
```

Comment:
I think we could remove sept/rept benchmark suite
as Ishan has a PR for removing sept,rept
https://github.com/OpenMined/PySyft/pull/6436
which would be merged in few and would break the benchmarks

Reply:
Thanks @rasswanth-s , I've removed it from the executor, for now, I'll phase out rept/sept suite entirely later. I'm making them a legacy code now.🙂